content/en/user-guide/aws/elastic-kubernetes-service/index.md

```diff
@@ -98,78 +135,44 @@ $ awslocal ecr create-repository --repository-name "fancier-nginx"
+
+To ensure the correct URL and port, it's important to use the `repositoryUrl` obtained from the `create-repository` request. This ensures that you have the accurate endpoint to access the repository.
 {{< /alert >}}
 
-Now let us pull the nginx image:
+You can now pull the `nginx` image from Docker Hub using the `docker` CLI:
+
 {{< command >}}
 $ docker pull nginx
 {{< / command >}}
-... tag it to our repository name:
+
+You can further tag the image to be pushed to ECR:
+
 {{< command >}}
 $ docker tag nginx localhost.localstack.cloud:4510/fancier-nginx
 {{< / command >}}
-... and push it to ECR:
+
+Finally, you can push the image to ECR:
+
 {{< command >}}
 $ docker push localhost.localstack.cloud:4510/fancier-nginx
 {{< / command >}}
 
 Now, let us set up the EKS cluster using the image pushed to local ECR.
-{{< command >}}
-$ awslocal eks create-cluster --name fancier-cluster --role-arn "arn:aws:iam::000000000000:role/eks-role" --resources-vpc-config "{}"
-{
-    "cluster": {
-        "name": "fancier-cluster",
-        "arn": "arn:aws:eks:us-east-1:000000000000:cluster/fancier-cluster",
-        "createdAt": "2022-04-13T16:38:24.850000+02:00",
-        "roleArn": "arn:aws:iam::000000000000:role/eks-role",
-        "resourcesVpcConfig": {},
-        "identity": {
-            "oidc": {
-                "issuer": "https://localhost.localstack.cloud/eks-oidc"
-            }
-        },
-        "status": "CREATING",
-        "clientRequestToken": "cbdf2bb6-fd3b-42b1-afe0-3c70980b5959"
-    }
-}
-{{< / command >}}
 
-Once the cluster status is "ACTIVE":
-{{< command >}}
-awslocal eks describe-cluster --name "fancier-cluster"
-{
-    "cluster": {
-        "name": "fancier-cluster",
-        "arn": "arn:aws:eks:us-east-1:000000000000:cluster/fancier-cluster",
-        "createdAt": "2022-04-13T17:12:39.738000+02:00",
-        "endpoint": "https://localhost.localstack.cloud:4511",
-        "roleArn": "arn:aws:iam::000000000000:role/eks-role",
-        "resourcesVpcConfig": {},
-        "identity": {
-            "oidc": {
-                "issuer": "https://localhost.localstack.cloud/eks-oidc"
-            }
-        },
-        "status": "ACTIVE",
-        "certificateAuthority": {
-            "data": "..."
-        },
-        "clientRequestToken": "d188f578-b353-416b-b309-5d8c76ecc4e2"
-    }
-}
-{{< / command >}}
+Next, we can configure `kubectl` to use the EKS cluster, using the [`UpdateKubeconfig`](https://docs.aws.amazon.com/eks/latest/APIReference/API_UpdateClusterConfig.html) API. Run the following command:
 
-... we will configure `kubectl`:
 {{< command >}}
-$ awslocal eks update-kubeconfig --name fancier-cluster && kubectl config use-context arn:aws:eks:us-east-1:000000000000:cluster/fancier-cluster
+$ awslocal eks update-kubeconfig \
```

Comment:
nit: line break seems a bit weird - maybe break up the command into two separate lines (each starting with `$`)..?